CCSA_ChatApplication/Controllers/MessageAndHistoryController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CCSA_ChatApp.Domain.DTOs.MessageDTOs;
+using CCSA_ChatApp.Infrastructure.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+
+namespace CCSA_ChatApplication.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MessageAndHistoryController : ControllerBase
```

Comment:
Change your controller name 
Maybe to message history

Reply:
Okay